test/AspNet.Security.OAuth.Providers.Tests/WechatHasUnionId/bundle.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
If you use the [Apple provider's tests](https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers/blob/dev/test/AspNet.Security.OAuth.Providers.Tests/Apple/AppleTests.cs) as a guide, the HTTP registrations can all be put in the existing Weixin bundle file.